packages/sdk-codegen/src/codeGen.ts

```diff
@@ -223,6 +223,10 @@ export abstract class CodeGen implements ICodeGen {
     return this.itself ? `${this.itself}.${value}` : value
   }
 
+  sortProperties(_a: IProperty, _b: IProperty) {
+    return 0
+  }
```

Comment:
Did you intend to leave this in? I realize it's not sorting anything right now, but I don't think we should be sorting properties for type declarations. This destroys any intentional grouping/order the developer may have created.

Reply:
yes, that was intended.